dali/test/python/test_operator_rotate.py

```diff
@@ -274,26 +317,23 @@ class RotatePerFrameParamsProvider(ParamsProvider):
     The expanded baseline pipeline must be provided with additional argument ``size``
     to make allowance for coalescing of inferred frames sizes
     """
-
     def __init__(self, input_params):
         super().__init__(input_params)
 
     def expand_params(self):
-        assert(self.num_expand == 1)
+        assert self.num_expand == 1
         expanded_params = super().expand_params()
         params_dict = {param_data.desc.name: param_data for param_data in expanded_params}
         expanded_angles = params_dict.get('angle')
         expanded_axis = params_dict.get('axis')
-        assert expanded_angles is not None and 'size' not in self.fixed_params and 'size' not in params_dict
-        sequence_extents = [
-          [sample.shape[0] for sample in input_batch]
-          for input_batch in self.input_data]
+        assert expanded_angles is not None and 'size' not in self.fixed_params and \
+            'size' not in params_dict
```

Comment:
```suggestion
        assert expanded_angles is not None and 'size' not in self.fixed_params \
               and 'size' not in params_dict
```
binary operators at the beginning of a continuation

Reply:
done